core/templates/services/UpgradedServices.ts

```diff
@@ -1188,7 +1188,11 @@ export class UpgradedServices {
     upgradedServices['PlaythroughIssuesBackendApiService'] =
       new PlaythroughIssuesBackendApiService(
         upgradedServices['HttpClient'],
-        upgradedServices['PlaythroughIssuesBackendApiService'],
+        upgradedServices['PlaythroughIssueObjectFactory'],
+        upgradedServices['UrlInterpolationService']);
+    upgradedServices['PlaythroughBackendApiService'] =
```

Comment:
Please check line 1185. `PlaythroughBackendApiService` is already declared above. Why are we declaring it here again?

Reply:
Great catch! Done

Reply:
:)